stock_picking_package_preparation_line/__init__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+##############################################################################
```

Comment:
Please replace the old headers with the [new short one](https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3)